tasks/__init__.py

```diff
@@ -958,6 +958,22 @@ def assets(ctx, dev=False, watch=False, colors=False):
 
 
 @task
+def spa_assets(ctx, production=False, watch=False):
+    """Build static assets for the OSF Ember SPA"""
```

Comment:
Note to self: this doesn't work at present, particularly because the utility needs to run in a separate folder.